docs/sql/01-query-optimizer.md

```diff
@@ -0,0 +1,689 @@
+# SQL Optimizer (state of 5.2.0)
```

Comment:
will we update the header in each release without major changes in the optimizer?

Reply:
Yes, we should.

Reply:
so in master we should have 5.3.0